changelog.d/20230705_164100_sylvain.baud.ext_iac_scan_parameters_fix.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- Moved the tempGGClient back to PyGitGuardian. This allow the scan parameters to be correctly processed when using iac scan diff
```

Comment:
I don't think this change requires a changelog entry: this is an implementation detail. It's not interesting for our users (does not mean it's not interesting to us!).

Reply:
Indeed, I removed the GGClient part as it is not interesting for our users.

However I detailed the `scan parameters` parts as this is a fix that can impact user. (Currently `minimum-severity` and `ignore-policy`are totally ignored in an `iac scan diff` )

Reply:
Thanks, at first I thought the changelog file could be removed, but this is nicer :+1:.